src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import duke.util.Constants;
+
+import java.util.List;
+import java.util.Arrays;
+
+public class Ui {
+
+    /**
+     * Prints a GAP added text for differentiating user and app texts.
+     *
+     * @param text Text to be printed.
+     */
+    public void print(String text) {
+        List<String> lines = Arrays.asList(text.split("\n"));
+        for (String line : lines) {
+            System.out.println(Constants.GAP + line);
+        }
+        System.out.println(Constants.LINE);
+    }
+
+    /**
+     * Prints initial greetings when the app starts.
+     */
+    public void initialGreet() {
```

Comment:
As per Java coding standard, you could change this method name into something that starts with a verb, for example "greetUser".